buidler.config.ts

```diff
@@ -29,6 +31,9 @@ const config: BuidlerConfig = {
     outDir: 'build/types',
     target: 'ethers-v5',
   },
+  gasReporter: {
+    currency: 'USD',
```

Comment:
pricing this in non-gas units seems like a very strange choice.

Reply:
it shows both gas units and usd price :p